packages/backend-for-frontend/src/config/config.ts

```diff
@@ -248,7 +249,8 @@ const BffProcessConfig = CommonHTTPServiceConfig.and(TenantProcessServerConfig)
   .and(SelfcareProcessConfig)
   .and(ClientAssertionValidationConfig)
   .and(EServiceTemplateS3Config)
-  .and(ApplicationAuditProducerConfig);
+  .and(ApplicationAuditProducerConfig)
+  .and(FeatureFlagAgreementApprovalPolicyUpdateConfig);
```

Comment:
Why this change is necessary? 
This change is part of https://github.com/pagopa/interop-be-monorepo/pull/1754 , if I'm not mistaken, you are not interested in this config and related endpoints in BFF.